tests/LogsActivityTest.php

```diff
@@ -9,206 +9,247 @@
 
 class LogsActivityTest extends TestCase
 {
-    /** @var \Spatie\Activitylog\Test\Article|\Spatie\Activitylog\Traits\LogsActivity */
-    protected $article;
+	/** @var \Spatie\Activitylog\Test\Article|\Spatie\Activitylog\Traits\LogsActivity */
```

Comment:
There seems to be something off with this diff. I can't easily see what changes you did make in regards to the PR. Could you only add lines and not modify existing ones that have nothing to do with this PR?